frontend/src/workbench/Site.tsx

```diff
@@ -98,63 +96,48 @@ const CustomCollapse = (props: CollapseProps) => {
   );
 };
 
-const DETAILS_PANEL_NAME = 'details';
-const REPRESENTATIONS_PANEL_NAME = 'representations';
-
-export const RightSite = ({ editingContextId, selection, setSelection, readOnly }: RightSiteProps) => {
+export const Site = ({ editingContextId, selection, setSelection, readOnly, contributions }: SiteProps) => {
```

Comment:
The code looks good, the only remaining things that should be improved is the ability to support more than two `WorkbenchView` per `Site`. We could probably use a vertical flex layout and say that the expanded accordion should take all the space available with `flex-grow: 1`

Reply:
Done